en/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: news_post
+title: "Ruby 2.6.0-preview1 Released"
+author: "naruse"
+translator:
+date: 2018-02-24 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.6.0-preview1.
+
+Ruby 2.6.0-preview1 is the first preview toward Ruby 2.6.0.
+This preview1 is released earlier than usual because it includes an important new feature, JIT.
+
+## JIT
+
+Ruby 2.6 introduces an initial implementation of JIT (Just-in-time) compiler.
+
+JIT compiler aims to improve performance of any Ruby program execution.
+Unlike ordinary JIT compilers for other languages, Ruby's JIT compiler does JIT compilation in a unique way, which prints C code to a disk and spawns common C compiler process to generate native code.
+See also: https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization
+
+How to use: Just specify "--jit" in command line or $RUBYOPT environment variable.
+Specifying "--jit-verbose=1" allows to print basic information of ongoing JIT compilation. See "ruby --help" for other options.
+
+The main purpose of this JIT release is to provide a chance to check if it works for your platform and to find out security risks before the 2.6 release.
+Currently JIT compiler is supported only when Ruby is built by gcc or clang and the compiler is available on runtime. Otherwise you can't use it for now.
+
+As of 2.6.0-preview1, we're just preparing infrastructure for JIT and very few optimizations are implemented.
+You can measure some of potential improvements in micro benchmarks with this release, but it is NOT ready for benchmarking final performance of Ruby's JIT compiler, especially for larger programs like Rails applications.
+
+We're going to implement method iniling in JIT compiler, which is expected to increase Ruby's performance in order of magnitude.
+Also, we're planning to increase the supported platforms, and the next plan is to support Visual Studio.
+
+Stay tuned for the new age of Ruby's performance.
+
+## New Features
+
+* Add Random.bytes. [Feature #4938]
+NOTE: 小粒な変更だけど賑やかしのために入れる。preview2ではけす
+
+* Add Binding#source_location.  [Feature #14230]
+  * This method returns the source location of binding, a 2-element array of `__FILE__` and `__LINE__`.  Traditionally, the same information could be retrieved by `eval("[__FILE__, __LINE__]", binding)`, but we are planning to change this behavior so that `Kernel#eval` ignores binding's source location [Bug #4352].  So, users should use this newly-introduced method instead of `Kernel#eval`.
+
+* Add :exception option to let Kernel.#system raise error instead of returning false. [Feature #14386]
+
+## Performance improvements
+
+* Speedup `Proc#call` because we dont' need to care about `$SAFE` any more.
```

Comment:
*don't*

Reply:
Thank you for pointing out! #1750 will fix it.